packages/ra-ui-materialui/src/button/Button.tsx

```diff
@@ -137,7 +141,9 @@ const StyledButton = styled(MuiButton, {
     },
 });
 
-const getLinkParams = (locationDescriptor?: LocationDescriptor | string) => {
+export const getLinkParams = (
```

Comment:
Why do you need to export it? I don't see it used anywhere else.

Reply:
Thanks I missed to add the behaviour in CreateButton

Reply:
Can you remove the `export` now?